res/controllers/Numark Mixtrack Pro.midi.xml

```diff
@@ -81,30 +81,57 @@ Modify by Darío José Freije 06/27/2012-->
             </control>
 
 <!-- CHANNEL 1 -->
-
-           <control>
-                <status>0xb0</status>
-                <midino>0x1b</midino>
-                <group>[Flanger]</group>
-                <key>lfoDepth</key>
+            <control>
+                <group>[EffectRack1_EffectUnit1_Effect1]</group>
+                <key>enabled</key>
+                <status>0x90</status>
+                <midino>0x63</midino>
                 <options>
-                    <diff/>
+                    <normal/>
                 </options>
             </control>
             <control>
-                <status>0xb0</status>
-                <midino>0x1c</midino>
-                <group>[Flanger]</group>
-                <key>lfoDelay</key>
+                <group>[EffectRack1_EffectUnit1_Effect1]</group>
+                <key>effect_selector</key>
+                <status>0xB0</status>
+                <midino>0x1B</midino>
+                <options>
+                    <selectknob/>
+                </options>
+            </control>
+            <control>
+                <group>[EffectRack1_EffectUnit1_Effect1]</group>
+                <key>meta</key>
+                <status>0xB0</status>
+                <midino>0x1C</midino>
                 <options>
                     <diff/>
                 </options>
             </control>
             <control>
-                <status>0xb0</status>
-                <midino>0x1d</midino>
+                <group>[EffectRack1_EffectUnit1_Effect1]</group>
+                <key>clear</key>
+                <status>0x90</status>
+                <midino>0x68</midino>
+                <options>
+                    <normal/>
+                </options>
+            </control>
+            <control>
                 <group>[Channel1]</group>
-                <key>pregain</key>
+                <key>rate</key>
+                <status>0xB0</status>
+                <midino>0x0D</midino>
+                <options>
+                    <normal/>
+                </options>
+            </control>
+            <control>
+                <group>[EffectRack1_EffectUnit1]</group>
+                <key>mix</key>
+                <description>MIDI Learned from 569 messages.</description>
```

Comment:
Please remove lines like this. They'll be displayed in the I/O tables and don't provide useful information.